### DISCUSSION TITLE
Nested folders for Hugo site

### DISCUSSION THREAD

**Discussion by secsonthebeach** [sveltia/sveltia-cms#227] [General]
Hello,
I was getting Decap CMS up and running for an established Hugo site. By using the beta feature for nested folders, I was able to view all of my content with Decap CMS.
At some point along the way, I discovered Sveltia CMS and like it much more! But I cannot seem to view all of my content in nested folders.
I’ve already tweaked my Hugo page bundles to follow the `Folder-named-for-page/_index` format throughout.

Also, as a side note, I observed a lot of yaml formatting errors in the browser console when switching to Sveltia. In a lot of cases I had to use Decap CMS to update those pages for the error to go away. I’m not exactly sure what was wrong with the formatting.

Please let me know what you think about the folders. I would much rather use Sveltia, it has a much nicer UI/UX!

**Comment by kyoshino** [comment]
Thanks for checking out Sveltia CMS! Nested Collections are not implemented yet and will be supported after the 1.0 release (sometime next year) as mentioned in the [README](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#current-limitations).

Nested Collections would make things highly complicated especially when i18n is involved, so I want to make sure everything is thoroughly tested. Decap CMS undoubtedly has bugs:

- https://github.com/decaporg/decap-cms/issues/4677
- https://github.com/decaporg/decap-cms/issues/6974

As for YAML formatting errors, what do these error messages say? Duplicate keys are possible culprits. Decap CMS probably doesn’t output a clear error because this issue is not fixed yet:

- https://github.com/decaporg/decap-cms/issues/7262

**Reply by secsonthebeach** [reply]
Thanks for your reply! Although it’s a bit disappointing. I’ll be looking forward to making the switch, Sveltia is really sleek!
I did not have duplicate keys, but I may have had a lot of duplicate values for differing keys. I was also making a transition from some TOML frontmatter to using all YAML, and switching some frontmatter keys to accepting lists of values.
If I notice errors again that are not my own doing, I’ll pass them along.

Thanks! Will be looking forward to nested colllections!

**Comment by josineto** [comment]
Just adding my use case for nested folders: I use Staticman for comments (uploaded to `data` folder, without moderation because my site doesn't receive a lot of comments), and with Netlify/Decap CMS I was able to use nested folders to view (and, more important, exclude) comment files through the CMS.
